spec/controllers/api/orders_controller_spec.rb

```diff
@@ -141,6 +161,16 @@ module Api
 
     private
 
+    def serialized_orders(orders)
+      serialized_orders = ActiveModel::ArraySerializer.new(
+        orders,
+        each_serializer: Api::Admin::OrderSerializer,
+        root: false
+      )
+
+      JSON.parse(serialized_orders.to_json)
```

Comment:
wouldn't to_json + parse be the same of doing nothing? just returning `serialized_orders`?

Reply:
No, I think we want an array of hashes instead of a serializer.

Reply:
Right, then [as_json](https://apidock.com/rails/v3.2.3/ActiveModel/Serializers/JSON/as_json) is what we need.

Reply:
I played around with this for too long, and this was the only way I could find to get a green test. Notes:

- Doing nothing gives us objects instead of the expected json data.
- Using `#as_json` doesn't give the same output as what's returned from the server (I did try it originally) so the test fails. It gives us a hash that has symbols for keys instead of strings. 
- Subsequently using `#stringify_keys!` on the hash results in a failure as well, because the hash has nested data. The recursive version `#deep_stringify_keys!` is not available until Rails 4. 

Any other suggestions?

Reply:
I'm okay with the current version.